src/components/Accounts/Transactions/Transaction/index.js

```diff
@@ -3,6 +3,28 @@ import moment from 'moment';
 import CategoryIcon from 'components/CategoryIcon';
 
 export default class Transaction extends React.Component {
+
+
+  static getTextualDecline(declinedReason){
+	if(!declinedReason) return false;
```

Comment:
Ah, apologies I wasn't clear. `if(!declinedReason) return false;` is written as one line whereas I prefer it to be written as:
```javascript
if (!declinedReason) {
  return false;
}
```
The blocks make the code easier to read.